service/Cargo.toml

```diff
@@ -24,6 +24,7 @@ paste = "1.0.8"
 poem = { version = "1.3.40", optional = true }
 regex = "1.6.0"
 rocket = { version = "0.5.0-rc.2", optional = true }
+serenity = { version = "0.11.5", default-features = false, features = ["client", "gateway", "rustls_backend", "model"], optional = true }
```

Comment:
Only the `client` feature is used by `shuttle_service`, right?

Reply:
One would think so, but I tried removing all but `client` and adding them back one by one. It actually won't compile without all of them. :thinking: